sound/soc/sof/ipc4-topology.h

```diff
@@ -30,6 +30,12 @@
 #define SOF_IPC4_NODE_INDEX(x)	((x) & 0xff)
 #define SOF_IPC4_NODE_TYPE(x)  ((x) << 8)
 
+/* Node ID for SSP type dai copiers */
+#define SOF_IPC4_NODE_INDEX_INTEL_SSP_IDX_MASK 0xF
+#define SOF_IPC4_NODE_INDEX_INTEL_SSP_IDX_SHIFT 4
```

Comment:
Can we drop the SHIFT define? Or is it going to be used some place?

While at it, I might drop the X from the IDX, or is it better to be short for 'index' instead of id?

Reply:
left up to me id drop both the mask and shift and use 0xF and 4 directly 
but the shift is used just below

Reply:
Sounds even better, if the mask is also only used via the macro

Reply:
done